src/main/java/org/betonquest/betonquest/compatibility/protocollib/conversation/MenuConvIO.java

```diff
@@ -253,25 +257,42 @@ private void start() {
     }
 
     private Location getBlockBelowPlayer(final Player player) {
```

Comment:
Please add some JavaDocs to this great algorithm so it can be understood more easily by people that do not know much about the MC block collision model!.